lib/src/configuration.cc

```diff
@@ -292,6 +307,8 @@ const Configuration::Agent& Configuration::getAgentConfiguration() const
 
 void Configuration::reopenLogfile() const
 {
+    // TODO(ale): consider reopening PCP access file
```

Comment:
I don't see the use of SIGUSR2 documented to re-open the logfile documented anywhere. I think it would be worth re-opening PCP access file on that signal, as well as documenting its use.